### PR TITLE
Add v2.13.0 Release Notes

### PR DESCRIPTION
**Pull request by nisgoel-amazon**
*Issue #1104, if available:*

*Description of changes:*
Update release notes for v2.13.0.

*CheckList:*
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/index-management/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by vikasvb90 — APPROVED** [review]

File: release-notes/opensearch-index-management.release-notes-2.13..0.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Version 2.13.0.0 2024-03-20
```
Comment (r1walz) [inline release-notes/opensearch-index-management.release-notes-2.13..0.0.md:1]: Typo in file name. It should be `opensearch-index-management.release-notes-2.13.0.0.md`.

File: release-notes/opensearch-index-management.release-notes-2.13..0.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Compatible with OpenSearch 2.13.0
+
+### Maintenance
+* Increment version to 2.13.0-SNAPSHOT. ([#1096](https://github.com/opensearch-project/index-management/pull/1096))
+* Upgrade com.github.seancfoley:ipaddress to mitigate CVE-2023-50570  ([#1126](https://github.com/opensearch-project/index-management/pull/1126))
+* Updates demo certs and admin keystore ([#1116](https://github.com/opensearch-project/index-management/pull/1116))
+*
```
Comment (r1walz) [inline release-notes/opensearch-index-management.release-notes-2.13..0.0.md:9]: Extra line item?

File: release-notes/opensearch-index-management.release-notes-2.13..0.0.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+### Bug fixes
+* Fix Typo in Alias Message ([#1137](https://github.com/opensearch-project/index-management/pull/1137))
```
Comment (r1walz) [inline release-notes/opensearch-index-management.release-notes-2.13..0.0.md:12]: Can you also add this PR in the `### Documentation` section? See #1092.

**Review by r1walz — CHANGES_REQUESTED** [review]
Thanks @nisgoel-amazon. Please, address the comments.

**Review by Hailong-am — APPROVED** [review]

**Pull request merged**